README.md

```diff
@@ -180,6 +180,17 @@ Once complete your base AWS resources should be in place
 
 If kubectl is unable to connect, the cluster is still starting, so wait a few minutes and try again; Terraform also creates new DNS entries, so you may need to flush your DNS cache. Once `cluster-info` returns Kubernetes master and KubeDNS your cluster is ready.
 
+### Helm RBAC setup
+
+Helm's Tiller should use its own service account. Create it like this:
+```
+kubectl create -f config/helm/tiller.yml
+# Tell helm to use it
+helm init --service-account helm
```

Comment:
This is *not* creating the `helm` service account, it's just telling tiller to use the service account just created with `kubectl create -f config/helm/tiller.yml` (above) right?

If that's the case, I think this should be `$ helm init --service-account helm` after the renaming to `tiller`.

Reply:
Do you mean it should be `helm init --service-account tiller`?

Reply:
I think so. If that's the service account created by `$ kubectl create -f config/helm/tiller.yml` we now changed the service account name to `tiller`, no? 

https://github.com/ministryofjustice/analytics-platform-ops/pull/175/files#diff-c5d78363155c97bdda457267956dc012R4

But I may be wrong.

Reply:
I've made this change: https://github.com/ministryofjustice/analytics-platform-ops/commit/9e36b7a532f5d54228cbc683e26cc3571d862eb5

Reply:
🏆